monai/apps/reconstruction/networks/nets/complex_unet.py

```diff
@@ -58,19 +62,23 @@ def __init__(
         dropout: Union[float, tuple] = 0.0,
         upsample: str = "deconv",
         pad_factor: int = 16,
+        conv_net: Optional[nn.Module] = None,
     ):
         super().__init__()
-        self.unet = BasicUNet(
-            spatial_dims=spatial_dims,
-            in_channels=2,
-            out_channels=2,
-            features=features,
-            act=act,
-            norm=norm,
-            bias=bias,
-            dropout=dropout,
-            upsample=upsample,
-        )
+        if conv_net is None:
+            self.unet = BasicUNet(
+                spatial_dims=spatial_dims,
+                in_channels=2,
+                out_channels=2,
+                features=features,
+                act=act,
+                norm=norm,
+                bias=bias,
+                dropout=dropout,
+                upsample=upsample,
+            )
+        else:
+            self.unet = conv_net
```

Comment:
If I understand it correctly, when conv_net is given, you will need to verify its `in_channels` and `out_channels` are 2

Reply:
good point!